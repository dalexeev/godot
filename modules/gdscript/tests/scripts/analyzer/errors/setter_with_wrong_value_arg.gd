var a: int: set = set_named

func test():
	pass

func set_named(name: StringName, value: Array) -> void:
	prints("set_named", name, value)
