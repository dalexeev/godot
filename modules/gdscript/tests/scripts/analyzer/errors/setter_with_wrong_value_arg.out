GDTEST_ANALYZER_ERROR
Function with value argument of type "Array" cannot be used as setter for a property of type "int".
